src/components/Navi.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import '../styles/Navi.css';
+
+const Navi = () => (
```

Comment:
- [ ] [OPTIONAL] Kindly reduce the `font-size` of the nav bar in order to improve the overall looking of your project.

Reply:
Yes sure, thank you 👍